.expeditor/config.yml

```diff
@@ -160,146 +162,4 @@ subscriptions:
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:libyajl2-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:cheffish-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:license-acceptance-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:ffi-libarchive-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:plist-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:ffi-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:net-ssh-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:tty-prompt-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:tty-screen-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:tty-table-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:pastel-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:erubis-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:bcrypt_pbkdf-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:ed25519-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:addressable-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:proxifier2-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:syslog-logger-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:uuidtools-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:iniparse-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:net-sftp-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:fauxhai-ng-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  # NOTE: The branch of Ohai here needs to be updated when setting up a stable branch of chef/chef
-  - workload: chef/ohai:main_completed:pull_request_merged:chef/ohai:main:*
-    actions:
-      - bash:.expeditor/update_bundler_dep.sh
-  # NOTE: When the stable branch of chef/chef is being cut you probably want to remove this subscription
-  - workload: chef/chefstyle:main_completed:pull_request_merged:chef/chefstyle:main:*
-    actions:
-      - bash:.expeditor/update_bundler_dep.sh
-  - workload: ruby_gem_published:train-rest-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:train-winrm-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: schedule_triggered:chef/chef:chef-18:adhoc_build_chef_18:*
-    actions:
-      - trigger_pipeline:validate/adhoc
```

Comment:
The entire block will stay as it is for chef-18 as well

Reply:
I see that the [PR](https://github.com/chef/chef/pull/9236) linked in the [how to doc](https://github.com/chef/chef/blob/main/docs/dev/how_to/bumping_the_major_version.md) has these removed but line mentioned in doc about updating this file `In ./expeditor/config.yml remove the update_dep.sh subscriptions which don't work against stable branches such as chefstyle and ohai.` does not seem to apply to these jobs as they are not against main branch of any of those gems. Referring [chef-17 to chef-18 branching changes ](https://github.com/chef/chef/commit/1ced14e4ef6d7cd6df8b27a9dc99374d49886e6a#diff-1b971b15e8bfa265b0b32a03d6e529c77681e5f0cf04c348e53081b5ad683f0eL242) seems there used to be such legacy code, which no longer exists. So these jobs are still needed to trigger PRs when any of those gems are released against base branch as `chef-18` (all of that is being taken care of by the script `expeditor/update_dep.sh ` itself using `EXPEDITOR_BRANCH` variable)